email_helper.user.js

```diff
@@ -0,0 +1,123 @@
+// ==UserScript==
+// @name         Tau Station Email Helper
+// @namespace    http://tampermonkey.net/
+// @downloadURL  https://github.com/taustation-fan/userscripts/raw/master/email_helper.user.js
+// @version      0.1
+// @description  Provides multi-email forwarding functionality to Tau Station email system
+// @author       Sergey Kudriavtsev <https://github.com/dot-sent>
+// @match        https://alpha.taustation.space/email/*
+// @grant        none
+// @require      https://code.jquery.com/jquery-3.3.1.min.js
+// @require      https://unpkg.com/turndown/dist/turndown.js
+// ==/UserScript==
+
+var localStorageKey = 'emailsToForward';
+
+var turndownService = new TurndownService();
+
+function extractEmailText(emailId) {
+    var emailBody = $('#email-body-' + emailId);
+    var emailHeader = $('#email-' + emailId);
```

Comment:
It is convention, though by no means a requirement, to name variables that contain jquery objects with a leading dollar sign, so `$emailBody` and `$emailHeader´.